src/test/java/bio/terra/service/auth/iam/IamServiceTest.java

```diff
@@ -96,4 +103,35 @@ public void testDeletePolicyMember() throws InterruptedException {
             eq(email));
     assertEquals(policyModel, result);
   }
+
+  @Test
+  public void testVerifyAuthorizations() throws Exception {
+    IamResourceType resourceType = IamResourceType.DATASET;
+    String id = ID.toString();
+
+    List<IamAction> hasActions = List.of(IamAction.MANAGE_SCHEMA, IamAction.READ_DATA);
+    when(iamProvider.listActions(eq(authenticatedUserRequest), eq(resourceType), eq(id)))
```

Comment:
I think `eq()` is only needed if you have any matcher arguments. If all arguments are constant (not matchers) then you don't need `eq()`
```suggestion
    when(iamProvider.listActions(authenticatedUserRequest, resourceType, id))
```

Reply:
Thanks! I made this change throughout this test class.